tools/FakeItEasy.Build/FakeItEasy.Build.csproj

```diff
@@ -0,0 +1,41 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Bullseye" Version="1.1.0-rc.2" />
+    <PackageReference Include="SimpleExec" Version="2.2.0" />
+    <PackageReference Include="GitVersion.CommandLine" Version="4.0.0-beta0012" ToolName="GitVersion" ToolExe="GitVersion.exe" />
+    <PackageReference Include="PdbGit" Version="3.0.41" ToolName="PdbGit" ToolExe="PdbGit.exe" />
+    <PackageReference Include="xunit.runner.console" Version="2.0.0" ToolName="XUnit" ToolExe="xunit.console.exe" />
+    <PackageReference Include="vswhere" Version="1.0.62" ToolName="VSWhere" ToolExe="vswhere.exe" />
+    <PackageReference Include="NuGet.CommandLine" Version="4.1.0" ToolName="NuGet" ToolExe="nuget.exe" />

```

Comment:
This rids us of downloading nuget.exe manually :wink:
(we should probably consider upgrading the version too, but I didn't want to make unrelated changes)